### PR TITLE
Add release notes for 0.245.1

### PR DESCRIPTION
**Pull request by yingsu00**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.245.rst [hunk 1]
```diff
@@ -4,6 +4,7 @@ Release 0.245
 
 .. warning::
     There is a bug causing failure at startup if function namespace manager is enabled and Thrift is not configured (:pr:`15501`).
+    There is a bug causing failure when reading ORC files having ARRAY/MAP/ROW of VARCHAR columns for some corner cases, and the fix is (:pr:`15549`).
```
Comment (bhhari) [inline presto-docs/src/main/sphinx/release/release-0.245.rst:7]: I would also mention `when using the SelectiveStreamReaders`

File: presto-docs/src/main/sphinx/release/release-0.245.1.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+===============
+Release 0.245.1
+===============
+
+.. warning::
+    There is a bug causing failure at startup if function namespace manager is enabled and Thrift is not configured (:pr:`15501`).
```
Comment (bhhari) [inline presto-docs/src/main/sphinx/release/release-0.245.1.rst:6]: I don't think we need to repeat this warning here

Comment (yingsu00) [reply presto-docs/src/main/sphinx/release/release-0.245.1.rst:6]: Really? As far as I can see in 0.245, this warning is still there.

Comment (yingsu00) [reply presto-docs/src/main/sphinx/release/release-0.245.1.rst:6]: 15501 is in 0.246 therefore I think we should keep the warning. @rongrong what do you think?

File: presto-docs/src/main/sphinx/release/release-0.245.rst (hunk 1 above)
Comment (yingsu00) [reply presto-docs/src/main/sphinx/release/release-0.245.rst:7]: @bhhari Thanks for reviewing! I think the users might not know the code `SelectiveStreamReaders`. How about saying "when Aria scan is enabled"?

File: presto-docs/src/main/sphinx/release/release-0.245.1.rst (hunk 2 above)
Comment (mayankgarg1990) [reply presto-docs/src/main/sphinx/release/release-0.245.1.rst:6]: I think we should keep it here

File: presto-docs/src/main/sphinx/release/release-0.245.rst (hunk 1 above)
Comment (mayankgarg1990) [reply presto-docs/src/main/sphinx/release/release-0.245.rst:7]: This is my suggestion (incorporating Ying's suggestion for Aria scan):

```
There is a bug causing failure when reading ORC files having ``ARRAY``/``MAP``/``ROW`` of ``VARCHAR`` columns for some corner cases when using Aria scan. (:pr:`15549`)
``

Comment (yingsu00) [reply presto-docs/src/main/sphinx/release/release-0.245.rst:7]: Updated to be
```
Fix a bug reading ORC files having ARRAY/MAP/ROW of VARCHAR columns using selective stream readers for some corner cases
```

Comment (yingsu00) [reply presto-docs/src/main/sphinx/release/release-0.245.rst:7]: @mayankgarg1990 Thank you fore reviewing! @bhhari thinks users may not know what Aria is, so I changed this to
```
There is a bug causing failure when reading ORC files with ``ARRAY``/``MAP``/``ROW`` of ``VARCHAR`` columns using the selective stream readers for some corner cases 
```
How do you like it?

Comment (mayankgarg1990) [reply presto-docs/src/main/sphinx/release/release-0.245.rst]: sounds reasonable

**Review by bhhari — APPROVED** [review]
LGTM

**Review by mayankgarg1990 — APPROVED** [review]

File: presto-docs/src/main/sphinx/release/release-0.245.1.rst (hunk 2 above)
Comment (rongrong) [reply presto-docs/src/main/sphinx/release/release-0.245.1.rst:6]: Ya, this won't be fixed until 246. We decided to not cherry-pick the fix for 245 to make sure we'd deploy 245 ontime.

File: presto-docs/src/main/sphinx/release/release-0.245.rst [hunk 3]
```diff
@@ -4,6 +4,7 @@ Release 0.245
 
 .. warning::
     There is a bug causing failure at startup if function namespace manager is enabled and Thrift is not configured (:pr:`15501`).
+    There is a bug causing failure when reading ORC files with ``ARRAY``/``MAP``/``ROW`` of ``VARCHAR`` columns using the selective stream readers for some corner cases (:pr:`15549`)
```
Comment (rongrong) [inline presto-docs/src/main/sphinx/release/release-0.245.rst:7]: I think we normally add separate warnings for different items (see 239 and 243 release notes). So add another `.. warning::` to this one.

**Review by rongrong — APPROVED** [review]

**Comment by yingsu00** [comment]
Thanks @bhhari @mayankgarg1990 @rongrong for reviewing. I'll wait for all tests passing before merging.

**Pull request merged**